why-promptlayer/evaluation-and-ranking.mdx

```diff
@@ -23,11 +23,11 @@ That means A/B testing is easy. Use the [Prompt Registry](/features/prompt-regis
 
 ## Scoring
 
-*Every request in PromptLayer has a “Score”. This score is an integer 0-100.*
+*Every request in PromptLayer has “Scores”. This scores are an integer 0-100.*
 
 <img src="/images/evaluation-score.png" />
 
-Ranking in PromptLayer revolves around this Score value. You can update it through the UI (see screenshot above) or programmatically ([More details here](/features/prompt-history/tracking-metadata-and-request-ids)).
+Ranking in PromptLayer revolves around this Score values. You can update the unnamed score through the UI (see screenshot above) or programmatically, which lets you define other named scores. ([More details here](/features/prompt-history/tracking-metadata-and-request-ids)).
```

Comment:
In PromptLayer, ranking is based on Score values. Scores can be updated via the UI or programmatically, allowing for the creation of named or unnamed scores. For further details, refer to the provided documentation on prompt history, metadata, and request IDs. ([More details here](/features/prompt-history/tracking-metadata-and-request-ids)).